index.html

```diff
@@ -0,0 +1,125 @@
+        <a class="underline" href="#">fullstackdeveloper@gmail.com</a>
+
+    </header>
+    <!-- Header Section End -->
+
+    <!-- Top Technologies Start -->
+    <section class="top-lang">
+        <h2>Top Languages</h2>
+        <div class="cards-area">
+            <div class="card">
+                <div class="card-body">
+                    <img src="./images/html-logo.webp" alt="Html-Logo">
+                    <h2>Html</h2>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Expedita, natus.</p>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-body">
+                    <img src="./images/css-logo.webp" alt="Html-Logo">
+                    <h2>CSS</h2>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Expedita, natus.</p>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-body">
+                    <img src="./images/javascript-logo.webp" alt="Html-Logo">
+                    <h2>JavaScript</h2>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Expedita, natus.</p>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-body">
+                    <img src="./images/bootstrap.png" alt="Html-Logo">
+                    <h2>Bootstrap</h2>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Expedita, natus.</p>
+                </div>
+            </div>
+        </div>
+        <div class="btn-join"><a href="#">Join Our Community</a></div>
+    </section>
+    <!-- Top Technologies End -->
+
+    <!-- Top Developers Section Start -->
+    <section class="top-develop" id="top-develop">
+        <h2 class="develop-title">Top Developers</h2>
+        <div class="top-area" id="top-area">
+        </div>
+        <div class="btn-more" id="btn-more">More <i class="fa-solid fa-angle-down"></i></div>
+    </section>
+    <!-- Top Developers Section End -->
+
+    <!-- Partner Section Start -->
+    <section class="partner">
+        <div class="container _container">
+            <h2 class="titleNumbre">Partners</h2>
+            <div class="numbersContent">
+                <span class="partner">Facebook</span>
+                <span class="partner">Google</span>
+                <span class="partner">Coinbase</span>
+                <span class="partner">NFT</span>
+                <span class="partner">crypto</span>
+            </div>
+        </div>
+    </section>
+    <!-- Partner Section End -->
+
+    <!-- Footer Section Start -->
+    <footer class="footer">
+        <div class="container _container">
+            <span class="logo-color"> <a href="./index.html" class="logo-color">Full-Stack </a></span>
+            <span>Sami Khan 2022 Some Right Reserved. </span>
+        </div>
```

Comment:
- [ ] In your desktop version kindly make agap between the two elements in the last footer so to be identical to the original design

- [ ] Kindly put your logo beside the copyright paragraph so since it is found in te origianl design the logo and the copyrights

- [ ] Kindly make the height the same as the origianl design so to be identical to the original ✅✅✅✅

- Your design
![image](https://user-images.githubusercontent.com/58553711/181280463-1219ae30-0e20-46a4-ae6d-528fdbff1f78.png)


- original design
![image](https://user-images.githubusercontent.com/58553711/181280587-34a66dcc-c548-47b5-add3-3d16ff809db9.png)